Saudox/app/Http/Controllers/PacienteController.php

```diff
@@ -16,7 +17,40 @@ public function __construct(){
 
     // Redireciona para a home do profissional
     public function home(){
-        return view('paciente.home');
+        $paciente = Paciente::find(Auth::id());
+
+        $ava_fono = $paciente->avaliacao_fono;
+        $ava_judo = $paciente->avaliacao_judo;
+        $ava_neur = $paciente->avaliacao_neuro;
+        $ava_tocp = $paciente->avaliacao_terapia_ocupacional;
+
+        $anamnese_fono = $paciente->anamnese__terapia__ocupacionals;
+        $anamnese_pnps = $paciente->anamneseFonoaudiologias;
+        $anamnese_tocp = $paciente->anamneseGigantePsicopedaNeuroPsicomotos();
```

Comment:
## Olá Carlos, a forma que foi implementado sobrecarrega a memória do servidor, pois você requisitou todas as informações de todas as avaliações, anamneses e futuramente evoluções. O servidor terá uma memória ram bem limitada e pode não chegar a suportar tante informação. Perceba que você não está utilizando as informações coletadas, essas estão sendo utilizadas apenas para verificar se o paciente tem anamnese. Sugiro fazer o seguinte:

```php
$id_ava_fono = $paciente->avaliacao_fono ? $paciente->avaliacao_fono['id'] : -1;
```
### E na view você verifica se o id é ou não -1, se for, é porque não possui avaliação de fonoaudiologia, e caso contrário, sabe-se que ele tem essa avaliação. Ademais, caso seja necessário o id para algum tipo de redirecionamento via get, você já o possui.

__Ademais, teste todos os casos antes de submeter, ok?__